packages/config/src/__tests__/loadConfigFromFile.test.ts

```diff
@@ -61,6 +61,7 @@ describe('loadConfigFromFile', () => {
       expect(resolvedPath).toMatch(path.join(ctx.fs.cwd(), 'prisma.config.ts'))
       expect(config).toMatchObject({
         experimental: true,
+        loadedFromFile: path.join(ctx.fs.cwd(), 'prisma.config.ts'),
```

Comment:
Just use:

```suggestion
          loadedFromFile: resolvedPath,
```

Reply:
Addressed in https://github.com/prisma/prisma/pull/26315